keyboards/m65/keymaps/default/readme.md

```diff
@@ -0,0 +1,16 @@
+![M65 Layout Image](https://i.imgur.com/NexMpvg.png)
+
+# Default M65 Layout
+
+This is the default layout for M65. For the most
+part it's a straightforward and easy to follow layout inspired by ISO with a UK turn.
+The only unusual key is the key in the upper left, which sends Escape normally, but Grave when in
+layer 1 aka LWR.
```

Comment:
Don't know about other editors, but I remember you mentioning that you (on occasions) use Vim. Doing `gqip` will reflow the text to `textwidth` here.